### PR TITLE
Rewrite SLES on POWER installation procedure

### PR DESCRIPTION
**Pull request by dmpop**
### Description
SLE on POWER installation procedure written from scratch based on input from Diego Assencio. This PR is for SLE15SP2 only.

### Checklist
* Check all items that apply.

*Are backports required?*.

- [ ] To maintenance/SLE15SP1
- [ ] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [ ] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

**Comment by dmpop** [comment]
PDF version for more convenient reviewing.

[book-sle-deployment_color_en.pdf](https://github.com/SUSE/doc-sle/files/4402895/book-sle-deployment_color_en.pdf)

File: xml/deployment_prep_power.xml [hunk 1]
```diff
@@ -25,16 +12,522 @@ fate#304108: PPC64 Hotplug memory remove
...
+      </para>
+    </step>
+    <step>
+      <para>
+	Select the desired &power; server and the LPAR. If a serial console session for the
+	chosen LPAR already exists, you need to close it first using the following command:
+      </para>
+      <screen>rmvterm -m <replaceable>SERVER</replaceable> -p <replaceable>LPAR</replaceable></screen>
+      <para>
+	Note that this command causes a hard reboot of the LPAR. To perform a soft reboot
```
Comment (dassencio) [inline xml/deployment_prep_power.xml]: There is some text missing here. The `rmvterm` command does not reboot the LPAR. The command which does is:

`chsysstate -r lpar -m <server-name> -o shutdown -n <lpar-name> --immed --restart`

File: xml/deployment_prep_power.xml [hunk 2]
```diff
@@ -25,16 +12,522 @@ fate#304108: PPC64 Hotplug memory remove
...
+       <textobject><phrase>Choosing a system role</phrase>
+       </textobject>
+      </mediaobject>
+     </informalfigure>
+    </step>
+    <step>
+      <para>
+	The next several screen allow you to specify the appropriate
+	time zone, and create a user. If you choose not to create a user,
+	you prompted to specify a root password.
```
Comment (dassencio) [inline xml/deployment_prep_power.xml]: "you **will be** prompted to"

File: xml/deployment_prep_power.xml (hunk 1 above)
Comment (dmpop) [reply xml/deployment_prep_power.xml:221]: Oops! Fixed.

File: xml/deployment_prep_power.xml [hunk 3]
```diff
@@ -25,16 +12,523 @@ fate#304108: PPC64 Hotplug memory remove
...
+     partitions manually. To avoid this, we recommend to have 10 GB reserved
+     for the root partition. Increase the minimum size to 16 GB if you plan
+     to enable Btrfs snapshots on the root volume (see <xref linkend="cha-snapper"/>).
+    </para>
+   </listitem>
+  </varlistentry>
+ </variablelist>
+ <para>
+   Before installing &productname;, make sure that the server has the latest
+   firmaware. For latest firmware, visit at IBM FixCentral (<link
```
Comment (lproven) [inline xml/deployment_prep_power.xml:177]:
```suggestion
   firmware. For latest firmware, visit at IBM FixCentral (<link
```

File: xml/deployment_prep_power.xml [hunk 4]
```diff
@@ -25,16 +12,523 @@ fate#304108: PPC64 Hotplug memory remove
...
+    <step>
+      <para>
+	The next several screen allow you to specify the appropriate
+	time zone, and create a user. If you choose not to create a user,
+	you are prompted to specify a root password.
+      </para>
+    </step>
+    <step>
+      <para>
+	In the installation summary screen, make sure to enable the SSH service
```
Comment (lproven) [inline xml/deployment_prep_power.xml:448]: Either "make sure the SSH service is enabled" **or** "enable the SSH service" but not both.

**Pull request merged**